streamlit_app.py

```diff
@@ -5,12 +5,54 @@
 import requests
 import en_core_web_sm
 import geocoder
+import base64
+import json
 
 # Load environment variables at the top to avoid errors
 load_dotenv()
 
 # I was not able to get the key to work loading from the env why?
 API_WEATHER_KEY = os.getenv("API_WEATHER_KEY")
+#Spotify API keys
+client_id = os.getenv("CLIENT_ID")
```

Comment:
`client_id` should be `spotify_client_id`. The same applies to `spotify_client_secret` and `spotify_refresh_token`